src/common/bisect_common.mli

```diff
@@ -65,6 +65,9 @@ val register_file :
 
 (** {1 [.coverage] output} *)
 
+val write_runtime_coverage : coverage -> out_channel -> unit
+(** Writes the [coverage] to the given output channel. *)
+
```

Comment:
This is not ideal. It expose a function only used in `src/report/merge.ml` but it was way easier to call inner function as we have a `coverage`. I believe the other solution would be to build _again_ the coverage using `Bisect_common.register_file`, but this seem too complicated for what we're trying to achieve here.